etc/proto/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG GOVERSION=golang:1.21.8
+ARG GOVERSION=golang:1.22.2
```

Comment:
Why is .circleci/* being upgraded to 1.21.9 but GOVERSION being set to 1.22.2 in these docker files?